pysite/route_manager.py

```diff
@@ -31,25 +38,27 @@ def __init__(self):
 
         # Load the main blueprint
         self.main_blueprint = Blueprint("main", __name__)
-        print(f"Loading Blueprint: {self.main_blueprint.name}")
+        self.log.debug(f"Loading Blueprint: {self.main_blueprint.name}")
         self.load_views(self.main_blueprint, "pysite/views/main")
         self.app.register_blueprint(self.main_blueprint)
-        print("")
+        self.log.debug("")
 
         # Load the subdomains
         self.subdomains = ['api', 'staff']
 
         for sub in self.subdomains:
             self.sub_blueprint = Blueprint(sub, __name__, subdomain=sub)
 
-            print(f"Loading Blueprint: {self.sub_blueprint.name}")
+            self.log.debug(f"Loading Blueprint: {self.sub_blueprint.name}")
             self.load_views(self.sub_blueprint, f"pysite/views/{sub}")
             self.app.register_blueprint(self.sub_blueprint)
-            print("")
+            self.log.debug("")
 
     def run(self):
         self.app.run(
-            port=int(os.environ.get("WEBPAGE_PORT", 8080)), debug="FLASK_DEBUG" in os.environ
+            host=os.environ.get("FLASK_HOST", "127.0.0.1"),
+            port=int(os.environ.get("WEBPAGE_PORT", 8080)),
+            debug="FLASK_DEBUG" in os.environ
```

Comment:
I don't understand this change, does it even belong in this PR?

Reply:
I actually don't fully understand this either. why wasn't host already here? and how did it work without host? and if it did work without it, why are we adding it?

Reply:
I needed to be able to override the default localhost value with 0.0.0.0 when running it in vagrant to be able to get to the site from my browser

Reply:
Ah, right, I getcha